test/setup.ts

```diff
@@ -21,20 +21,17 @@ export const setupRecording = (): Context =>
     adapters: ["node-http"],
     matchRequestsBy: {
       headers: {
-        exclude: ["authorization", "host", "content-length"],
-      },
-      url: {
-        hash: false,
-        hostname: false,
-        password: false,
-        pathname: false,
-        port: false,
-        protocol: false,
-        query: false,
-        username: false,
+        exclude: ["host", "content-length"],
```

Comment:
Why remove authorization header? was it conflicting with tests?

Reply:
I was checking the failing tests and forgot to add it back.